src/test/java/api/support/fakes/FakeStorageModule.java

```diff
@@ -181,7 +182,7 @@ private void create(RoutingContext routingContext) {
 
     originalBody.put("id", id);
 
-    preProcessBody(originalBody).thenAccept(body -> {
+    preProcessBody(new JsonObject(), originalBody).thenAccept(body -> {
```

Comment:
hmm... I just think, that maybe it better to use null here and then check on null in the pre-processors... 

Because there is no old value for insert operation, but `new JsonObject()` means that the old value is an empty object.

Reply:
Yes, I will do it.